content/sensu-go/6.6/web-ui/search.md

```diff
@@ -18,6 +18,29 @@ When you apply a search to a web UI page, it creates a unique link for the page
 You can bookmark these links and share your favorite search combinations.
 You can also [save your favorite searches][8].
 
+## Events and entities search limits
+
+In Sensu Go 6.6.3 and subsequent versions, web UI search queries on the events and entities pages will stop after returning a certain number of matches.
+Without these limits, the search operation can diminish cluster health.
+
+{{% notice note %}}
+**NOTE**: The search limits on events and entities do not apply to Sensu Go versions prior to 6.6.3.
+{{% /notice %}}
+
+### Events search limit
+
+On the events page, if you use etcd for event storage, search queries will stop after returning a certain number of events:
```

Comment:
maybe we can move "if you use etcd for event storage" above, as it relates to both subheadings

Reply:
I decided to add it in all three places to make sure readers see it even if they only read one section here.